bindings/cs/MongoDB.Libmongocrypt.Test32/MongoDB.Libmongocrypt.Test32.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
+        <TargetFrameworks>net472;netcoreapp2.1</TargetFrameworks>
         <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp2.2</TargetFrameworks>
```

Comment:
Shouldn't this be `net472;netcoreapp2.1;netcoreapp3.0` same as MongoDB.Libmongocrypt.Test.csproj? And non-Windows `netcoreapp2.1;netcoreapp3.0`?

Reply:
this test project is tricky (and I don't remember details in the top of my head), but regardless it, I think, no reason to avoid adding the whole list of supported TFs

Reply:
done

Reply:
Alright.